cmd/prchecker/main_test.go

```diff
@@ -0,0 +1,232 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/google/go-github/github"
+)
+
+func TestStringAllowsModuleSpan(t *testing.T) {
+	var tests = []struct {
+		body          string
+		matchExpected bool
+	}{
+		{"foo", false},
+		{"don't ALLOW_MODULE_SPAN", false},
+		{"ALLOW_MODULE_SPAN", true},       // PR body is just exception
+		{"> ALLOW_MODULE_SPAN", true},     // support quoting exception
+		{"\t ALLOW_MODULE_SPAN\t ", true}, // support whitespace padding
+		{"ALLOW_MODULE_SPAN\nat start of file", true},
+		{"at end of file\nALLOW_MODULE_SPAN", true},
+	}
+
+	for _, tt := range tests {
+		testname := fmt.Sprintf("does %v allow spanning %v", tt.body, tt.matchExpected)
+
+		t.Run(testname, func(t *testing.T) {
```

Comment:
Usually we add a `name` field in the struct that defines each testcase. It allows for providing better names that actually describes why we are testing a particular case.